README.md

```diff
@@ -17,6 +17,10 @@ This package implements type-safe memory units. It pursues the following goals:
 1. Focus on correctness.
 2. Low amount of boilerplate should be required to use the library.
 
+The ideas behind this package are described in the following blog post:
+
+ * [Insane in the Membrain](https://kowainik.github.io/posts/membrain)
```

Comment:
Do you think that here is the nice place to put it?

Reply:
I think this is an excellent place to put it :+1: